rules/run_binary.bzl

```diff
@@ -20,6 +20,22 @@ Runs a binary as a build action. This rule does not require Bash (unlike native.
 
 load("//lib:dicts.bzl", "dicts")
 
+_NOOPS = ("$(rlocationpath", "$(rootpath")
+_EXPANDS = ("$(execpath", "$(location")
+
+def _is_expandable(value):
```

Comment:
This check is misleading; if the template contains both `$(location ...)` and `$(rlocationpath ...)`, *both* will be expanded.

I think we ought to allow people to expand `$(rlocationpath` etc. - I don't see a use case for it, but going out of our way to filter it out complicates the code for no good reason.